api/Blob.json

```diff
@@ -0,0 +1,271 @@
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "type": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Blob/type",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "5"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "4"
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "11"
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "5.1"
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "slice": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Blob/slice",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": [
+              {
+                "version_added": "21"
+              },
+              {
+                "version_added": "5",
+                "version_removed": "21",
+                "prefix": "webkit",
+                "notes": "The <code>slice()</code> method had initially taken <code>length</code> as the second argument to indicate the number of bytes to include in the new <code>Blob</code>. If you specified values such that <code>start + length</code> exceeded the size of the source <code>Blob</code>, the returned <code>Blob</code> contained data from the start index to the end of the source <code>Blob</code>.&nbsp;That version of the <code>slice()</code> was implemented in <a href='https://hg.mozilla.org/mozilla-central/rev/1b3947ed93c6'>Firefox 4</a>, <a href='http://trac.webkit.org/changeset/55670'>WebKit</a>, and <a href='http://www.opera.com/docs/specs/presto28/file/#blob'>Opera 11.10</a>. However, since that syntax is differed from <a href='/en-US/JavaScript/Reference/Global_Objects/Array/slice'><code>Array.slice()</code></a> and <a href='/en-US/JavaScript/Reference/Global_Objects/String/slice'><code>String.slice()</code></a>, Gecko and WebKit removed support and added support for the new syntax as <code>Blob.slice()</code>/<a href='http://trac.webkit.org/changeset/83873'><code>Blob.webkitSlice</code></a>."
```

Comment:
Let's remove these lengthy notes here, too. They are way too long and we have the situation explained in the wiki at the top of the page already. The important (machine-readable) info here are the prefixes.